donner.go

```diff
@@ -105,15 +106,21 @@ func execCommand(cfg *Cfg, params []string) error {
 	}
 
 	cmd := exec.Command(execHandler.BaseCommand, cliArgs...)
-
-	out, err := cmd.CombinedOutput()
-	if err != nil {
-		return err
+	cmd.Stdout = os.Stdout
+	cmd.Stdin = os.Stdin
+	cmd.Stderr = os.Stderr
+
+	var waitStatus syscall.WaitStatus
+	if err := cmd.Run(); err != nil {
+		_, _ = os.Stderr.WriteString(fmt.Sprintf("Error: %s\n", err.Error()))
+		if exitError, ok := err.(*exec.ExitError); ok {
+			waitStatus = exitError.Sys().(syscall.WaitStatus)
+			os.Exit(waitStatus.ExitStatus())
+		}
+	} else {
+		os.Exit(waitStatus.ExitStatus())
```

Comment:
This doesn't make sense to me. Shouldn't we be looking at `cmd.ProcessState` to determine if the command finished successfully / error'd out? I think we can vastly simplify this snippet unless I'm overseeing something...